src/network/networkd-network.c

```diff
@@ -305,6 +306,10 @@ int network_verify(Network *network) {
                 if (section_is_invalid(fdb->section))
                         fdb_entry_free(fdb);
 
+        LIST_FOREACH_SAFE(static_mdb_entries, mdb, mdb_next, network->static_mdb_entries)
+                if (section_is_invalid(mdb->section))
```

Comment:
Please introduce `mdb_entry_verify()` or something, and check that group multicast address is configured.